Remotion/Web/Web.Development.WebTesting.IntegrationTests.RequireUI/ScreenshotTest.cs

```diff
@@ -0,0 +1,103 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using System.Drawing;
+using Coypu;
+using NUnit.Framework;
+using Remotion.Web.Development.WebTesting.IntegrationTests.Infrastructure.ScreenshotCreation;
+using Remotion.Web.Development.WebTesting.PageObjects;
+using Remotion.Web.Development.WebTesting.ScreenshotCreation;
+using Remotion.Web.Development.WebTesting.ScreenshotCreation.Annotations;
+using Remotion.Web.Development.WebTesting.ScreenshotCreation.Fluent;
+using Remotion.Web.Development.WebTesting.WebDriver;
+
+namespace Remotion.Web.Development.WebTesting.IntegrationTests.RequireUI
+{
+  [TestFixture]
+  public class ScreenshotTest : IntegrationTest
+  {
+    /// <summary>
+    /// Tests that the <see cref="ScreenshotTooltipAnnotation"/> is drawn correctly.
+    /// </summary>
+    [Category("Screenshot")]
+    [Test]
+    public void DrawTooltip ()
+    {
+      ScreenshotTestingDelegate<ElementScope> test = (builder, target) =>
+      {
+        Helper.BrowserConfiguration.BrowserAnnotateHelper.DrawTooltip(builder, target, GetTooltipStyleForCurrentBrowser());
+
+        builder.Crop(target.ForElementScopeScreenshot(), new WebPadding(30, 60, 30, 60));
+      };
+
+      var home = Start();
+      var element = home.Scope.FindId("tooltipTarget");
+      Helper.RunScreenshotTestExact<ElementScope, ScreenshotTest>(element, ScreenshotTestingType.Both, test);
+    }
+
```

Comment:
I think there are two tests missing here? We have 4 deleted methods in the other project but only two of them show up here.